### ISSUE TITLE
PPPoE support

### ISSUE THREAD

**Issue by luoshuai123** [PcapDotNet/Pcap.Net#79]
please,it is helpful.

**Comment by beppe9000** [comment]
That's true. Beside that it is awful how there is no open-source PPPoE server program for Windows...

**Comment by rmcavalcante** [comment]
Hello. 
PPPoE support for windows is really useful.
Thank you
